includes/class-wc-skroutz-analytics-product.php

```diff
@@ -148,4 +183,34 @@ private function get_product_terms_suffix() {
 
 		return $terms;
 	}
+
+	/**
+	 * Get the suffix of the id that we send to sku reviews widget. The suffix contains for every variation
```

Comment:
I would replace `send` with `use`

Reply:
Also I would rephrase "The suffix contains for every variation the first term id" to "The suffix is the first term of every grouped attribute in settings"